index.bs

```diff
@@ -10202,6 +10269,21 @@ the <code>typeof</code> operator will return "function" when applied to an inter
 </div>
 
 
+<h4 id="legacy-window-alias">Legacy window alias</h4>
```

Comment:
I'd move that whole section up in to the introductory section of §3.6 either as second or third ¶ and turn it into something like this:

```
If the [{{LegacyWindowAlias}}] extended attribute was specified on an interface,
then for each [=identifier=] mentioned in the extended attribute,
there must be a corresponding property on the [=primary global interface=],
known as the <dfn id="dfn-legacy-window-alias" export>legacy window alias</dfn>.
The name of the property is the [=legacy window alias=]'s [=identifier=],
and its value is a reference to the [=interface object=] for the [=interface=].
The property has attributes
{ \[[Writable]]: <emu-val>true</emu-val>, \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val> }.
```